Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -1649,6 +1649,25 @@ static float currentBacklightLevel()
     userInstalledFontFamilyMap() = WTFMove(fontFamilyMap);
 }
 
+#if ENABLE(INITIALIZE_ACCESSIBILITY_ON_DEMAND)
+void WebProcess::initializeAccessibility(Vector<SandboxExtension::Handle>&& handles)
```

Comment:
Do you think we could add an assertion to make sure this is never called more than once?